### ISSUE TITLE
ABE: Related Work

### ISSUE THREAD

**Issue by countvajhula** [drym-org/old-abe#1]
Members of the community,
If you know of any works or ideas that seem to you to be related to this project, please comment below.

Any reported related work will be considered in the process of Dialectical Inheritance Attribution (DIA) to determine:

- is it related to the present work in a meaningful way for the purposes of DIA?
- is the work entitled to a share in the revenues from the present project?
- is the present project entitled to a share in the revenues from that work?
- are they both entitled to a share in each other's revenues?

Any reports (themselves) of related work that are accepted by the arbitrators as related represent acts of value and are recognizable as such in an ABE system.

**Comment by willhansen** [comment]
I tried to come up with a software license that worked along these lines a few years back, but it never went anywhere: https://github.com/willhansen/RoyaltyOffRevenueLicense 

I ran across another related project a bit more recently, but the main dev vanished: https://github.com/openfare/openfare

**Comment by countvajhula** [comment]
This is awesome, thank you for reporting these @willhansen . Your "Royalty off revenue" license is just the kind of thing we're doing with ABE. Openfare also looks very relevant and it's not impossible that we could use it or something like it to help manage the actual payments, which are currently done manually by "trustees" / repo admins.

How did you find out about ABE, by the way? If you're interested, we [meet regularly](https://calendar.google.com/calendar/u/0/embed?src=4a464a65d09d47948cfb879659b77f56c293b588990ae718302530412265589b@group.calendar.google.com) -- we'd love to have you join. Take care.

**Comment by willhansen** [comment]
I ran into your [EmacsConf 2022 talk](https://www.youtube.com/watch?v=1WIulCmIHZ4) and really liked the social approach to this problem, and the git-based accounting system.  

 I like the social approach because that's what tripped up the the Royalty Off Revenue license.  Turns out leading with "hey, you should be paid for this work you're already used to doing for free for higher ideals" isn't a great sell.  Too much business perspective rather than open-source-community perspective went into that approach.

Having transparent accounting with accumulating amounts owed seems just a lot easier and standard than the crypto-currency approach of OpenFare.  None of the most-of-the-projects-using-the-technology-are-scams baggage of crypto-currency either.  Might be more viable in a few years, once the technology matures a bit and the scams die out.

Do you have any publicly visible forums set up? (maybe discussions on this repo?)

Looking forward to your RacketCon talk.

**Comment by countvajhula** [comment]
Yes, 100%. That is a very clear analysis. I might use it as a testimonial somewhere 😊 

Re: forums, we don't have one yet and will probably will need to set one up at some point. I'd like to consider other options besides GitHub for this so as not to be overly tied to any particular infrastructure, but free is a primary criterion at this time. Any other ideas besides Discussions? If not, it might just make sense to start there, and probably on the [foundation](https://github.com/drym-org/foundation) repo. Thank you for the suggestion.

There is the public [ABE mailing list](https://groups.google.com/g/abe-congress) you should definitely join. That's mostly used for announcements and soliciting participation in things like DIA, but not so much for discussion. There are going to be some exciting announcements on there soon.
